lambda_functions/process/make_vector_tiles/index.js

```diff
@@ -71,6 +73,44 @@ async function readGeojsonFiles(localDir) {
   return geojson;
 }
 
+
+async function emptyS3TilesDir(uuid, type) {
+  var S3 = new AWS.S3();
+  var keepRunning = true;
+  var params = {
+    Bucket: process.env.S3_BUCKET,
+    Prefix: `campaigns/${uuid}/render/${type}/tiles/`
+  };
+  var deletionsNumber = 0;
+
```

Comment:
🙌  promises and es6. Let's replace `var` with `const`/`let`, so we can use consistent es6.